requirements/requirements.txt

```diff
@@ -1,4 +1,3 @@
-ovos-plugin-manager~=0.0.4
-SpeechRecognition~=3.8
-neon_utils~=0.6
-requests~=2.26
\ No newline at end of file
+ovos-plugin-manager
```

Comment:
Are there version conflicts with something here? Unpinned dependencies open up the possibilities for upgrading to an incompatible version of a package and long-running dependency resolution that could download every version of each of these packages to try and find a comparible one

Reply:
The change was made in [V0.0.11a4](https://github.com/OpenVoiceOS/OVOS-plugin-manager/tree/V0.0.11a4).  I think I took care of it.